src/Game/zScript.h

```diff
@@ -1,6 +1,25 @@
 #ifndef ZSCRIPT_H
 #define ZSCRIPT_H
 
+#include <types.h>
+#include "../Core/x/xBase.h"
+#include "../Core/x/xDynAsset.h"
 
+struct xScriptAsset : xDynAsset
+{
+    // There is definitely more here, but idk what.
+};
```

Comment:
It's in the dwarf data.. https://github.com/pslehisl/bfbbdecomp/blob/master/dwarf/SB/Game/zScript.cpp#L1106-L1110

Please don't make any types definitions that are partially filled out. We shouldn't really ever have any comments like this.